### ISSUE TITLE
tango.core.Vararg does not compile with current LDC

### ISSUE THREAD

**Issue by n8sh** [SiegeLord/Tango-D2#109]
As of at least ldc2 version 1.24.0 the import `ldc.vararg` does not exist.

Demonstration:
```d
#!/usr/bin/env dub --single --compiler=ldc2
/+dub.sdl:
dependency "tango" version="~>1.0.5+2.0.85"
+/
import tango.core.Vararg;
void main()
{
}
```
Trying to run the above file fails with:
```console
.../tango/core/Vararg.d(43,19): Error: module vararg is in file 'ldc/vararg.d' which cannot be read
```

**Comment by n8sh** [comment]
Fixed by #100. No release tagged since then though.

**Comment by SiegeLord** [comment]
I tagged a new release.
